intl/msg_hash_es.h

```diff
@@ -1,3 +1,131 @@
+MSG_HASH(
+	MENU_ENUM_LABEL_COLLECTION,
+	"collection"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_CONFIGURATIONS,
+	"configurations"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_CONFIGURATION_SETTINGS,
+	"Determina cómo se cargan y priorizan los archivos de configuración"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_FAVORITES,
+	"favorites"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_HELP,
+	"help"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_INFORMATION,
+	"information"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_LOAD_STATE,
+	"loadstate"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_NETPLAY,
+	"netplay"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_NETPLAY_LAN_SCAN_SETTINGS,
+	"Busca y se conecta a anfitriones netplat en la red local."
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_ONLINE,
+	"online"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_REBOOT,
+	"reboot"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_RESUME,
+	"resume"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_RUN,
+	"collection"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_SAVE_STATE,
+	"savestate"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_SETTINGS,
+	"settings"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_SHUTDOWN,
+	"shutdown"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_THUMBNAILS,
+	"thumbnails"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_UNDO_LOAD_STATE,
+	"undoloadstate"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_UNDO_SAVE_STATE,
+	"undosavestate"
+	)
+MSG_HASH(
+	MSG_DEVICE_CONFIGURED_IN_PORT,
+	"configurado en puerto"
+	)
+MSG_HASH(
+	MSG_FAILED_TO_SET_DISK,
+	"Fallo al establecer disco"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_OFF,
+	"off"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_ON,
+	"on"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_DISABLED,
+	"disabled"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_ENABLED,
+	"enabled"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_SECONDS,
+	"seconds"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_STATUS,
+	"status"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_FILTER_BY_CURRENT_CORE,
+	"filter_by_current_Core"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_AUDIO_RESAMPLER_DRIVER_SINC,
+	"sinc"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_AUDIO_RESAMPLER_DRIVER_CC,
+	"cc"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_INPUT_DRIVER_UDEV,
+	"udev"
+	)
+MSG_HASH(
+	MENU_ENUM_LABEL_INPUT_DRIVER_LINUXRAW,
+	"linuxraw"
+	)
```

Comment:
All the above 32 hashes don't appear in msg_hash_us.h, where did they come from? 
This breaks my method of diff against msg_hash_us.h, i have to remove them to get them to sync correctly

Reply:
Actually these lines are from intl/msg_hash_lbl.h

I will fix the most obvious problems (the ones you commented on) and force push

I'm a bit shot of time right now but I have a longer WIP reply to the remaining points (one example of this are these 32 literals). Will post them in another PR hopefully soon.